src/sidebar/store/modules/test/annotations-test.js

```diff
@@ -24,19 +23,6 @@ function createTestStore() {
 // in the tests for the whole Redux store
 
 describe('sidebar/store/modules/annotations', function() {
-  let fakeDefaultPermissions;
```

Comment:
Running into woes mocking these utils, and recent direction suggests I do not try to mock them...

Reply:
What issues did you run into?

Reply:
Irrelevant now as this change has been removed from this branch because of a regression—will address when I return to these changes.